NvidiaGPU/Nvidia-GPU-Linux-Resources.json

```diff
@@ -0,0 +1,677 @@
+{
+    "Metadata": {
+        "Version": 20240829,
+        "Vendor": "NVIDIA",
+        "AcceleratorType": "GPU",
+        "OS": "Linux",
+        "DefaultSelectionsPerVMFamily": {
+            "NC series": "CUDA",
+            "NCv2 series": "CUDA",
+            "NCv3 series": "CUDA",
+            "NC_A100_v4 series": "CUDA",
+            "NCasT4_v3 sizes": "CUDA",
+            "NVv3 Series": "GRID",
+            "NVadsA10_v5 Series": "GRID"
+        },
+        "Comment": "We install the default set of drivers as mentioned in the DefaultSelectionsPerVMFamily for each VM family. If the user wants to install GRID driver on NC or ND series, can do so by passing runtime settings on supported GPU VM families. For checking the supported VM families for a particular driver category, refer to SupportedVmFamilyAndGPUModel under each category below. For installing GRID driver on NC Series, please specify {'installGridNC':True} in the runtime setting. For installing GRID driver on ND series, please specify {'installGridND':True} in the runtime settings. Runtime settings can be provided from Azure CLI - (az vm extension set  --resource-group <rg-name> --vm-name <vm-name> --name NvidiaGpuDriverLinux --publisher Microsoft.HpcCompute --settings \"{'installGridNC': True}\""
+    },
+    "Latest": {
+        "Category": [
+            {
+                "Name": "GRID",
+                "Versions": [
+                    {
+                        "DriverVersion": "535.161",
+                        "vGPUVersion": "16.5",
+                        "SupportedVmFamilyAndGPUModel": {
+                            "NCasT4_v3 sizes": "Nvidia Tesla T4",
+                            "NVv3 Series": "NVIDIA Tesla M60",
+                            "NVadsA10_v5 Series": "NVIDIA A10"
+                        },
+                        "Driver": [
+                            {
+                                "SupportedOSVersions": [
+                                    "Ubuntu 22.04 LTS",
+                                    "Ubuntu 20.04 LTS",
+                                    "Ubuntu 18.04 LTS",
+                                    "Red Hat Enterprise Linux 8.6, 8.8, 8.9",
```

Comment:
Would it be better if Red Hat Enterprise Linux 8.6, 8.8, 8.9 is separate into three lines so that the format is consistent?

I.e. 
Red Hat Enterprise Linux 8.6
Red Hat Enterprise Linux 8.8
Red Hat Enterprise Linux 8.9